datasets/ccaligned_multilingual/README.md

```diff
@@ -0,0 +1,321 @@
+```
+
+An instance of `sentences` type:
+
+```
+{'LASER_similarity': 1.2734256982803345, 'Source_Sentence': '>>> PhD Students in 2018', 'Target_Sentence': '>>> လုၵ်ႈႁဵၼ်းၸၼ်ႉ Ph.D ၼႂ်းပီ 2018', 'from_english': True}
+```
+
+### Data Fields
+
+For `documents` type:
+
+- `Domain`: a `string` feature containing the domain.
+- `Source_URL`: a `string` feature containing the source URL.
+- `Source_Content`: a `string` feature containing the content on Source_URL.
+- `Target_URL`: a `string` feature containing the target URL.
+- `Target_Content`: a `string` feature containing the content on Target_URL.
```

Comment:
The `Source_Content` and `Target_Content ` fields are now replaced by a field `translation` with two subfields.
One subfield is `en_XX` and the other one is the other language code.